### DISCUSSION TITLE
"data owner" versus "data provider"

### DISCUSSION THREAD

**Discussion by robbiemorrison** [cogent-computing/COPED#26] [General Chat]
I noticed at a recent panel discussion that the Copyright Office of the European Commission used the phrase "data provider" rather than "data owner" in relation to this kind of technical data.  I think that approach is quite useful because the thresholds for effort (under UK copyright law) and investment (under database protection law) are both legally fuzzy and hard for external parties to determine.  So the notion of "data provider" takes those issues off the table.&nbsp;R

**Comment by jbrusey** [comment]
Seems reasonable. Would you like to put together a pull request for the associated wording change?

**Comment by robbiemorrison** [comment]
Is there a wording change required?  No hits for "data owner"in this repo by my reckoning.  Perhaps this should be recorded as a guideline instead?

**Reply by jbrusey** [reply]
understood. I'm not sure where to record that other than this discussion though!

**Comment by colinstephen** [comment]
Hi Robbie, thanks for this. @SteveHaben is planning to pick up suggestions/discussions such as yours from GitHub, to pipe into future planning via the soon-to-be-convened CoPED working group (let him know if you'd like to contribute to that). All suggestions greatly appreciated!

**Reply by robbiemorrison** [reply]
I will contact @SteveHaben next week then.  Thanks.

**Reply by SteveHaben** [reply]
Thanks Robbie, interesting suggestion. We should keep note of this.
